packages/terra-form-input/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-form-input",
+  "main": "lib/FormInput.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-form-input",
+    "FormInput",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.0.0"
```

Comment:
we will need to update terra-base to the latest when it's released soon

Reply:
I'll add it to my todo list.

Reply:
Done b7fb7ce2ada572d12750eb100d869c1e3cc3137b